manager/controllers/app/solve_test.go

```diff
@@ -510,6 +510,70 @@ func TestWriteNewAsset(t *testing.T) {
 	g.Expect(solution.DataPath[0].Module.Name).To(gomega.Equal(writeModule.Name))
 }
 
+// This test checks the write scenario for a new asset.
+// Storage account is for MySQL, module supports S3 only.
+// Result: failure to create the data path
+func TestStorageAndModuleMismatch(t *testing.T) {
+	t.Parallel()
+	g := gomega.NewGomegaWithT(t)
```

Comment:
The 2 new tests are not going to pass with `USE_CSP=True` until I push my changes.
Maybe leave this file as-is, and I will commit the modified file with my changes?

Reply:
OK

Reply:
@zivnevo I've added the check for USE_CSP set to true, in which case the tests won't run.